File: wrappers/dotnet/indy-sdk-dotnet/PoolApi/InvalidPoolException.cs
namespace Hyperledger.Indy.PoolApi
{
    /// <summary>
    /// Exception thrown when attempting to use a pool that has been closed or is invalid.
    /// </summary>
    public class InvalidPoolException : IndyException
    {
        /// <summary>
        /// Initializes a new PoolClosedException.
        /// </summary>
        internal InvalidPoolException(string message) : base(message, (int)ErrorCode.PoolLedgerInvalidPoolHandle)
        {

        }
    }

}

File: wrappers/java/src/main/java/com/evernym/vdrtools/pool/InvalidPoolException.java
package com.evernym.vdrtools.pool;

import com.evernym.vdrtools.ErrorCode;
import com.evernym.vdrtools.IndyException;

/**
 * Exception thrown when attempting to use a poll that has already been closed.
 */
public class InvalidPoolException extends IndyException
{
	private static final long serialVersionUID = 7124250084655044699L;
	private final static String message = "The pool is closed or invalid and cannot be used.";

	/**
	 * Initializes a new PoolClosedException.
	 */
	public InvalidPoolException()
	{
		super(message, ErrorCode.PoolLedgerInvalidPoolHandle.value());
	}
}